scripts/dts/gen_defines.py

```diff
@@ -166,6 +167,21 @@ def relativize(path):
         return path
 
 
+def write_deps(node):
+    # Writes dependency ordinal information for the node.
+
+    def dep_ords(nodes):
+        return [node.dep_ordinal for node in nodes]
+
+    out_node(node, 'ORDINAL', node.dep_ordinal)
+    out_node_init(node, 'REQUIRES_NODES', dep_ords(node.depends_on))
+    out_node_init(node, 'REQUIRES_DEVS',
+                  dep_ords([n for n in node.depends_on if n.is_device]))
```

Comment:
Maybe `dep` or the like instead of `n`. Not sure what `n` stands for.

Reply:
On second thought, this isn't enough to capture parameterized dependencies; I'll rethink this. I guess we'll need to special-case by type of dependency (clock subsystem, interrupt controller, GPIO spec, etc.)